python/tvm/relay/_parser.py

```diff
@@ -410,6 +440,14 @@ def visitFuncType(self, ctx):
 
         return ty.FuncType(arg_types, ret_type, [], None)
 
+    def visitGraphExpr(self, ctx):
+        graph_nid = int(ctx.LOCAL_VAR().getText()[1:])
```

Comment:
this will fail with an internal python message if the user accidentally uses a CNAME